bubuku/cli.py

```diff
@@ -2,22 +2,39 @@
 
 import click
 
-from bubuku.amazon import Amazon
 from bubuku.config import load_config, KafkaProperties, Config
+from bubuku.env_provider import EnvProvider
 from bubuku.features.remote_exec import RemoteCommandExecutorCheck
-from bubuku.features.swap_partitions import load_swap_data
-from bubuku.id_generator import get_broker_id_policy
 from bubuku.zookeeper import load_exhibitor_proxy, BukuExhibitor
-from bubuku.zookeeper.exhibior import AWSExhibitorAddressProvider
 
 _LOG = logging.getLogger('bubuku.cli')
 
 
-def __get_opt_broker_id(broker_id: str, config: Config, zk: BukuExhibitor, amazon: Amazon) -> str:
+def _print_table(table: list, print_function=None):
```

Comment:
I'd rather use existing package https://pypi.python.org/pypi/PrettyTable but it is more kind of suggestion.